functions/microwin/Invoke-Microwin.ps1

```diff
@@ -76,6 +76,7 @@ public class PowerManagement {
     }
 
     $imgVersion = (Get-WindowsImage -ImagePath $mountDir\sources\install.wim -Index $index).Version
+    Write-Host "The Windows Image Build Version is: $imgVersion"
```

Comment:
```suggestion
    Write-Host "Version of selected image: $imgVersion"
```

While this works correctly, I would rephrase this a bit

Reply:
OK. It is a good change, regardless

Reply:
I do agree it could probably do with a rewording from my phrasing, best one would be a something that translates build numbers into actual version numbers, build such and such is Win11 23H2 for instance, but then you have to maintain that for future changes, so not ideal...